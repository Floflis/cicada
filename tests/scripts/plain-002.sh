ulimit -n 999
ulimit -n
ulimit -n 888
ulimit -n
ulimit -S -c 256
ulimit -H -c 1024
ulimit -c -H
ulimit -c
