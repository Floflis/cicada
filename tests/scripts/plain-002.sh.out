999
888
1024
256
